bears/js/PrettierLintBear.py

```diff
@@ -0,0 +1,33 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='prettier',
+        use_stdout=True,
+        use_stderr=True)
+class PrettierLintBear:
+    """
+    Formats JavaScript files according to the opinionated code format
+    using ``prettier``.
+
+    See <https://github.com/prettier/prettier> for more information.
+    """
+    LANGUAGES = {'JavaScript'}
+    REQUIREMENTS = {NpmRequirement('prettier', '1')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Consistency', 'Correctness', 'Whitespace',
+                  'Parentheses', 'Strings', 'Empty lines',
+                  'Multi-line objects'}
+    SEE_MORE = 'https://prettier.io/'
+
+    def process_output(self, output, filename, file):
+        stdout, stderr = output
+        regex = '(?P<message>.*)'
+        yield from self.process_output_corrected(stdout, filename, file)
+        yield from self.process_output_regex(stderr, filename, file, regex)
+
+    @classmethod
+    def create_arguments(cls, filename, file, config_file):
```

Comment:
@Nshul what is the reason that you're making `create_arguments` a `classmethod`.

Reply:
so as to use class name when required ;P. In case of change in class name many things arent to be changed

Reply:
no need to make this a class method as of now ;P

Reply:
Other Linter Bears were using class method so I made it this way. If i change it to static method then also the implementation remains same, but i guess having a class method might be better for any kind of changes later.

Reply:
be minimalistic, make it a `staticmethod` :+1: In case someone needs it later he/she will change it to a classmethod then :+1: